Validation/EcalDigis/plugins/EcalSelectiveReadoutValidation.cc

```diff
@@ -558,16 +558,18 @@ void EcalSelectiveReadoutValidation::analyzeEB(const edm::Event& event, const ed
   bool eventError = false;
   nEbZsErrors_ = 0;
   nEbZsErrorsType1_ = 0;
-  vector<pair<int, int> > xtalEtaPhi;
+  vector<pair<int, int>> xtalEtaPhi;
 
   xtalEtaPhi.reserve(nEbPhi * nEbEta);
+  std::unique_ptr<std::array<std::array<energiesEb_t, nEbPhi>, nEbEta>> ebEnergies =
```

Comment:
Why `ebEnergies` has to be re-defined here locally, when it is already defined in the header?
Asymmetry wrt `eeEnergies`

Reply:
It seems a bug had crept in.  Fixed it now. Thanks.